jobs/nginx/spec

```diff
@@ -4,49 +4,50 @@ packages:
 - nginx
 
 templates:
-  bin/nginx_ctl:     bin/nginx_ctl
+  bin/ctl:     bin/ctl
   config/nginx.conf: config/nginx.conf
   config/mime.types: config/mime.types
   config/ssl_key:    config/ssl_key
   config/ssl_crt:    config/ssl_crt
 
+consumes:
+- name: shield-daemon
+  type: shield-daemon
+
 properties:
-  shield.daemon.port:
+  port:
```

Comment:
I'd like to get it to just `port` as 443, assume https, and stop providing `http_port`.

Or if https is disabled, then `port` would be explicitly `80`.

@geofffranks @frodenas ideas/preference to moving to single `https`/`port: 443` property?